activerecord/lib/active_record/callbacks.rb

```diff
@@ -460,7 +460,7 @@ def _create_record
     end
 
     def _update_record
-      _run_update_callbacks { super }
+      _run_update_callbacks { _update_record { super } }
```

Comment:
I get what this is doing after reading it a few times, but it is pretty confusing at first glance because the method being called has the same method name. I wonder if it's worth refactoring this and calling a more appropriately named method explicitly instead of relying on `super`.

Reply:
Hmm I agree. What do you think about this?
https://github.com/rails/rails/compare/a25a55b1ac11a7de8ca8584e278d78ee26f63d35..ba69cd2d2130cf6d42bd201da9b67ebd756ec4a4